lib/core/styles/_index.scss

```diff
@@ -70,4 +70,5 @@
     @include adf-sidenav-layout-theme($theme);
     @include adf-clipboard-theme($theme);
     @include adf-snackbar-theme($theme);
+    @include mat-expansion-panel-theme-get-823-fix($theme);
```

Comment:
can we have normal names that don't incorporate tickets, or any other references? what does 823 mean and are you sure everyone will know what it means in the future?

Reply:
I wanted to point out that this change will no longer be needed once an upgrade to @angular/material 8.2.3 is done - probably that was not the best way to do it... 
I've renamed that mixin and updated the AC of ADF's 'Upgrade angular material to version 8' task instead: https://issues.alfresco.com/jira/browse/ADF-4702